docs/RT1.md

```diff
@@ -0,0 +1,220 @@
+# Raviart-Thomas 1 using barycentric coordinates
+
```

Comment:
nice document! one small comment, in the local machine the arrays are represented with the brackets correctly. however, on github they are not render well. Maybe you can try with the bmatrix instead. Not sure if it works 100%